HTML/EN/plugins/Spotty/settings/basic.html

```
@@ -121,6 +121,48 @@
+		[% END %]
+	[% END %]
+
+	[% WRAPPER setting title="PLUGIN_SPOTTY_ADVANCED_SETTING" desc="" %]
+		[% "PLUGIN_SPOTTY_ADVANCED_SETTING_DESC" | string %]
+	[% END %]
+
+	[% IF canConnect; WRAPPER setting title="" desc="" %]
+		<input name="pref_optimizePreBuffer" id="pref_optimizePreBuffer" type="checkbox" [% IF prefs.pref_optimizePreBuffer %] checked [% END %]>
+		<label for="pref_optimizePreBuffer">[% "PLUGIN_SPOTTY_OPTIMIZE_PRE_BUFFERING" | string %]</label>
+	[% END; ELSE %]
+		<input name="pref_optimizePreBuffer" id="pref_optimizePreBuffer" type="hidden" value="[% prefs.pref_optimizePreBuffer %]">
```

HTMLHint (1.9.2)

[error] The id value [ pref_optimizePreBuffer ] must be unique.

(id-unique)